test/JDBC/expected/col_length-vu-prepare.out

```diff
@@ -0,0 +1,230 @@
+GO
+
+CREATE PROCEDURE col_length_prepare_p6 AS (SELECT COL_LENGTH('sys_column_length_test_table', 'col_sql_variant'));
+GO
+
+CREATE PROCEDURE col_length_prepare_p7 AS (SELECT COL_LENGTH('sys_column_length_test_table', 'col_datetimeoffset'));
+GO
+
+CREATE PROCEDURE col_length_prepare_p8 AS (SELECT COL_LENGTH('sys_column_length_test_table', 'col_uniqueidentifier'));
+GO
+
+-- Invalid column, should return NULL
+CREATE PROCEDURE col_length_prepare_p9 AS (SELECT COL_LENGTH('sys_column_length_test_table', 'test_col'));
+GO
+
+-- Invalid table, should return NULL
+CREATE PROCEDURE col_length_prepare_p10 AS (SELECT COL_LENGTH('sys_column_length_test_table_invalid', 'col_char'));
+GO
+
+CREATE FUNCTION col_length_prepare_f1()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_xml'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f2()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_varcharmax'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f3()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_nvarcharmax'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f4()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_varbinarymax'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f5()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_bit'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f6()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_tinyint'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f7()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_bigint'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f8()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_smallint'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f9()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_smallmoney'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f10()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_money'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f11()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_smalldatetime'));
+END
+GO
+
+CREATE FUNCTION col_length_prepare_f12()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 'col_real'));
+END
+GO
+
+-- Invalid column, should return NULL
+CREATE FUNCTION col_length_prepare_f13()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', 1));
+END
+GO
+
+-- Invalid column, should return NULL
+CREATE FUNCTION col_length_prepare_f14()
+RETURNS INT AS
+BEGIN
+RETURN (SELECT COL_LENGTH('sys_column_length_test_table', -1));
+END
+GO
+
+-- Invalid table, should return NULL
+CREATE FUNCTION col_length_prepare_f15()
+RETURNS INT AS
+BEGIN
```

Comment:
You can add few having empty text as parameters , using some cast in parameters and also using this function in some tables and as expressions.